regions/core/core.py

```diff
@@ -161,6 +161,36 @@ def to_shapely(self):
         """
         raise NotImplementedError("")
 
+    @abc.abstractmethod
+    def make_patch(self, **kwargs):
```

Comment:
Sorry, I missed that... I've commented it on it :) If we do decide to change it, it's only a search and replace at least? 😬